src/model/database/db_build.sql

```diff
@@ -0,0 +1,49 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users,articles,contacts,chats CASCADE;
+
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY ,
+	user_name VARCHAR(100) NOT NULL,
+	user_email TEXT NOT NULL UNIQUE,
+	user_password TEXT,
+	permission  VARCHAR(100) DEFAULT 'user'
+);
+INSERT INTO users (user_name,user_email,user_password,permission) VALUES
+('salwa','salwa@hotmail.com','123','admin');
+
+INSERT INTO users (user_name,user_email,user_password,permission) VALUES
+('ons','ons@hotmail.com','1861997','doctor'),
+('kannan','kannan@hotmail.com','rami','doctor'),
+('mohanned','mohanned@hotmail.com','fofo','doctor');
+
+CREATE TABLE articles (
+article_id SERIAL PRIMARY KEY ,
+article_title VARCHAR(250) NOT NULL,
+article_img TEXT ,
```

Comment:
Is it allowed to have an article without image if not please add a NOT NULL constraint 
If it is allowed then I would advice you to have a default image to keep the good design of your page and add the constraint 
Or you could handle it with js by checking the existance of the img and add a default link before rendering the page